openedx/core/djangoapps/user_api/views.py

```diff
@@ -921,10 +923,11 @@ def _apply_third_party_auth_overrides(self, request, form_desc):
                 if current_provider:
                     # Override username / email / full name
                     field_overrides = current_provider.get_register_form_data(
-                        running_pipeline.get('kwargs')
+                        running_pipeline.get('kwargs'),
+                        current_provider=current_provider
                     )
 
-                    for field_name in self.DEFAULT_FIELDS:
+                    for field_name in self.DEFAULT_FIELDS + self.EXTRA_TPA_FIELDS:
```

Comment:
I'm not sure if this is the right construct to get this information. Why do we need to know that these fields are "TPA fields"? Why not just iterate through all the fields for the form, while being aware that we might not get a value?